webapp/channels/src/packages/mattermost-redux/src/selectors/entities/files.ts

```diff
@@ -46,6 +47,12 @@ export function makeGetFilesForPost(): (state: GlobalState, postId: string) => F
     );
 }
 
+export function getFilesForEditHistory(state: GlobalState, editHistoryPost: Post): FileInfo[] {
+    const useLocal = getCurrentUserLocale(state);
+    const fileInfos = editHistoryPost?.metadata?.files ?[...editHistoryPost.metadata.files] : [];
+    return sortFileInfos(fileInfos, useLocal);
+}
```

Comment:
Since this receives the state, I guess this is a selector. It returns a new list on every call, so it is a bad selector :P

I guess we can consider using a `createSelector` so the same input will always return the same output.

Reply:
Done.